src/_includes/macros/filters.html

```diff
@@ -230,6 +230,9 @@
                     {{ _filter_checkbox('calendar', 'Richard Cordray')|safe }}
                 </label>
                 <label class="form-group_item">
+                    {{ _filter_checkbox('calendar', 'Meredith Fuchs')|safe }}
```

Comment:
Should there be spaces around the pipe here and elsewhere? I saw a similar change in the Refresh PR (have we standardized this yet?)